From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Tue, 16 Aug 2011 17:36:49 +0200
Subject: [PATCH] fbgrab: include zlib.h

Newer libpng versions do not include zlib.h in their api any more, so if
we use symbols from there, we should include zlib.h directly.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 fbgrab.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/fbgrab.c b/fbgrab.c
index fc89f5a..0c959f6 100644
--- a/fbgrab.c
+++ b/fbgrab.c
@@ -23,6 +23,7 @@
 #include <getopt.h>
 #include <sys/vt.h>   /* to handle vt changing */
 #include <png.h>      /* PNG lib */
+#include <zlib.h>     /* zlib */
 #include <linux/fb.h> /* to handle framebuffer ioctls */
 
 #define	VERSION	"1.0 beta 1"
